script/Jenkinsfile.fossa

```diff
@@ -0,0 +1,30 @@
+hubCred = [
+    $class: 'UsernamePasswordMultiBinding',
```

Comment:
$class is legacy syntax to be avoided. 
Also group credential binding within a single `withCredentials` definition

prefer: 
```
withCredentials([
    usernamePassword(credentialsId: 'orcaeng-hub.docker.com', usernameVariable: 'REGISTRY_USERNAME', passwordVariable: 'REGISTRY_PASSWORD'),
    string(credentialsId: 'fossa-api-key', variable: 'FOSSA_API_KEY')
]) { ... }